docs/requirements_for_rtd.txt

```diff
@@ -1,12 +1,21 @@
 Flask-Cache==0.13.1
+Flask-Cache==0.13.1
+Flask-Cors==3.0.1
```

Comment:
perhaps one should make this a set rather than a list

Reply:
Apparently the update script is buggy and duplicated the lines, doesn't hurt though

Reply:
yes, here: https://github.com/aiidateam/aiida_core/blob/f6e28cef645d18e9bb08932f6b3399bb96d319e1/docs/update_req_for_rtd.py#L33
make it a `set` instead of a `list`

Reply:
Ah, my bad, didn't realize you were referencing the scripts itself. I will push a fix soon